static/app/views/releases/detail/overview/deploys.tsx

```diff
@@ -19,21 +20,21 @@ type Props = {
 const Deploys = ({version, orgSlug, projectId, deploys}: Props) => {
   return (
     <Wrapper>
-      <SectionHeading>{t('Deploys')}</SectionHeading>
-
-      {deploys.map(deploy => (
-        <Row key={deploy.id}>
-          <StyledDeployBadge
-            deploy={deploy}
-            orgSlug={orgSlug}
-            version={version}
-            projectId={projectId}
-          />
-          <TextOverflow>
-            <TimeSince date={deploy.dateFinished} />
-          </TextOverflow>
-        </Row>
-      ))}
+      <SidebarSection title={t('Deploys')}>
```

Comment:
Not sure if we should use SidebarSection component, it's adding a line next to title that's not in the design.